### ISSUE TITLE
Remove sensitive info from log

### ISSUE THREAD

**Issue by lionkor** [Lucky4Luuk/beammp_rust_server#6]
The log must **not** contain:

- Player IPs
- Server auth key
- Player public or private keys

**Comment by Lucky4Luuk** [comment]
Agreed 100%, these are currently here for debugging purposes. I'll remove the unneeded ones asap.
